pandas/core/nanops.py

```diff
@@ -109,6 +109,8 @@ def f(values, axis=None, skipna=True, **kwds):
             try:
                 if values.size == 0 and kwds.get('min_count') is None:
                     # We are empty, returning NA for our type
```

Comment:
add a TODO here (and maybe create an issue for this)? (can change later)

Reply:
You mean for having all the `nan*` methods just correctly handle empty inputs?

Reply:
yes, for your 1, 2, 3 options. we don't need to change this now, can be done in 0.23